bench/asv_bench/README.md

```diff
@@ -0,0 +1,25 @@
+# xDSL Benchmarks
+
+These benchmarks leverage [Airspeed Velocity](https://asv.readthedocs.io/en/stable/using.html).
+
+## Usage
+
+To perform the benchmark on your machine:
+
+``` sh
```

Comment:
```suggestion
``` sh
pip install -r requirements-optional.txt
```

Reply:
Actually asv creates its own environment for each run, so this won't do anything

Reply:
You can run `asv dev` to reuse the current environment